hawkular-inventory-feature-pack/src/main/xsl/subsystem-templates/hawkular-inventory-logging.xsl

```diff
@@ -18,47 +18,31 @@
 
 -->
 
-<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:xalan="http://xml.apache.org/xalan" version="2.0">
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:xalan="http://xml.apache.org/xalan" version="2.0" exclude-result-prefixes="xalan">
 
   <xsl:output method="xml" version="1.0" encoding="UTF-8" indent="yes" xalan:indent-amount="4" standalone="no" />
-  <xsl:strip-space elements="*" />
 
   <!-- //*[local-name()='config']/*[local-name()='supplement' and @name='default'] is an xPath's 1.0
        way of saying of xPath's 2.0 prefix-less selector //*:config/*:supplement[@name='default']  -->
   <xsl:template match="//*[local-name()='config']/*[local-name()='supplement' and @name='default']/*[local-name()='replacement' and @placeholder='LOGGERS']">
     <xsl:copy>
       <xsl:apply-templates select="@*|node()"/>
-      <xsl:element name="logger" namespace="{namespace-uri()}">
-        <xsl:attribute name="category">org.hawkular.inventory</xsl:attribute>
-        <xsl:element name="level">
-          <xsl:attribute name="name">
-            <xsl:text disable-output-escaping="yes">${hawkular.log.inventory:INFO}</xsl:text>
-          </xsl:attribute>
-        </xsl:element>
-      </xsl:element>
-      <xsl:element name="logger" namespace="{namespace-uri()}">
-        <xsl:attribute name="category">org.hawkular.inventory.rest.requests</xsl:attribute>
-        <xsl:element name="level">
-          <xsl:attribute name="name">
-            <xsl:text disable-output-escaping="yes">${hawkular.log.inventory.rest.requests:INFO}</xsl:text>
-          </xsl:attribute>
-        </xsl:element>
-      </xsl:element>
-    </xsl:copy>
-  </xsl:template>
-
-  <xsl:template
-      match="//*[local-name()='config']/*[local-name()='subsystem']/*[local-name()='console-handler']/*[local-name()='level']">
-    <xsl:copy>
-      <xsl:apply-templates select="@*|node()"/>
-      <xsl:attribute name="name">DEBUG</xsl:attribute>
+      <logger category="org.hawkular.inventory">
```

Comment:
> Have you checked the output of this? In the past, I've seen this being translated into the standalone.xml with some superfluous (or even incorrect) xml namespace declarations. Which is why this was originally done the more verbose way using xsl.

Yes, the output is correct. `exclude-result-prefixes="xalan"` on L21 does the trick.